config/config.md

```diff
@@ -296,6 +296,7 @@
 | `store_addrs` | Array | -- | Store server address default to etcd store. |
 | `store_key_prefix` | String | `""` | If it's not empty, the metasrv will store all data with this key prefix. |
 | `backend` | String | `EtcdStore` | The datastore for meta server. |
+| `pg_table_name` | String | `greptime_metakv` | Table name in postgresql to store metadata. Only effect when backend is `PostgresStore`. |
```

Comment:
I think the `pg` prefix is not proper, because we may use other RDMS in the future. Rename it into `meta_table_name`?